outplacement_order_interpreter/static/src/js/activity.js

```diff
@@ -32,11 +32,14 @@ var Activity = MailActivity.include({
             // If we are trying to remove an interpreter booking ask the user
             // if all things the user have to do before removing it has been done.
             let dialog = new Dialog(this, {'title': 'Cancel Interpreter',
-                                           '$content': $('<p>Contact Tolkportalen and use reference:' + activity.interpreter_booking_ref + ' to cancel Interpreter bookings <br> Only press Yes after confirmation.</p>'),
+                                           '$content': $('<p>
+                                                         <b>Avboka tolkbokningen</b> <br>
+                                                         Avboka tolken genom att skicka ett epostmeddelande till <a href="mailto:team-crm@arbetsformedlingen.se?subject=Avboka tolk med referens ' + activity.interpreter_booking_ref + '.">team-crm@arbetsformedlingen.se och angereferensnummer ' + activity.interpreter_booking_ref + '.
```

Comment:
I suggest writing in english in the code and providing, then translating it in the translation file. Also "angereferensnummer"